geometry_msgs/Pose[] poses
---
bool success
string message

